src/proc/occlusion-filter.cpp

```diff
@@ -6,10 +6,17 @@
 
 #include "proc/synthetic-stream.h"
 #include "proc/occlusion-filter.h"
+#include  "../../common/tiny-profiler.h"
+#include <vector>
+#include <cmath>
+
+#define ROTATION_BUFFER_SIZE 8
+#define VERTICAL_SCAN_WINDOW_SIZE 64
+#define DEPTH_OCCLUSION_THRESHOLD 1
```

Comment:
Note that depth buffer is in depth units (uint16_t), not meters, so having threshold = 1 is super restrictive. 
Need to convert from meters to depth units and use say 50cm as the threshold.

Reply:
depth units can be extracted from depth frame

Reply:
I changed the units as you suggested and found that ideal threshold is 5cm